PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/springcloud/ui/SpringCloudAppSettingPanel.java

```diff
@@ -179,35 +182,41 @@ private void init() {
         cbSubscription.addItem(CommonConst.REFRESH_TEXT);
     }
 
-    private void initCluster(String initialValue) {
```

Comment:
do we need to use isClusterInitialized to guard the initialization? it is only called from ctor.

Reply:
got it, since a prev PR change the calling place.